src/xdocs/config_coding.xml

```diff
@@ -464,14 +464,37 @@ public class A {
           more readable if it comes after the last <code>case</code>.
         </p>
       </subsection>
-
+      <subsection name="Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+          </tr>
+          <tr>
+            <td>skipIfLastAndSharedWithCase</td>
+            <td>whether to allow <code>default</code> along with case and not last</td>
```

Comment:
`and not last` -> `if they are not last`.
Please add an xdoc java examples of new violation, and non-violation case.